src/htmlexporter.cpp

```diff
@@ -327,47 +327,10 @@ void HTMLExporter::exportBasket(BasketScene *basket, bool isSubBasket)
     stream << QString(
         "  </div>\n"
         "  <p class=\"credits\">%1</p>\n").arg(
-            i18n("Made with <a href=\"http://basket.kde.org/\">%1</a> %2, a KDE tool to take notes and keep information at hand.",
+            i18n("Made with <a href=\"http://launchpad.net/basket\">%1</a> %2, a kdelibs-based tool to take notes and keep information at hand.",
```

Comment:
I removed that commit, so you can push this.
Related question: can I push another review which removed "KDE"? BasKet works everywhere; the old meaning of "KDE" referred there was about the desktop environment, which has not been the meaning of KDE for a long time. Also, it is not developed inside of  the KDE Community (and in that case the proper wording would be "a tool by KDE").